### ISSUE TITLE
how to delete an index when removing it from main DB

### ISSUE THREAD

**Issue by Blackops404** [elasticquent/Elasticquent#129]
i want to delete an index of a post when removing post from mysql

**Comment by Pashaster12** [comment]
As I understood, You want to remove the post from ElasticSearch index after deleting it from DB? If it is so, you should run YourEloquentModel::reindex(); for updating the index's data. If you want to delete index completely, run YourEloquentModel::deleteIndex();

**Comment by Blackops404** [comment]
@Pashaster12 tnx man, but is it ok ? reindex after every delete a record from db.

**Comment by Pashaster12** [comment]
@Blackops404 No, it's not ok))) The best solution is make a cron job with the reindexation which will be run with the periodicity that you want. 5 minute will be ok.

**Comment by Blackops404** [comment]
@Pashaster12 tnx again,  how can i delete a document using this Elasticquent, when i want to use deleteDocuments() method laravel says :Call to undefined method

**Comment by Pashaster12** [comment]
@Blackops404 it's ok, because there is no such function in Elasticquent.

**Comment by Blackops404** [comment]
@Pashaster12 thanks helping me ;) i have final question : when i want insert arabic texts sometimes,below error is occured, i have no idea why it's happening, i tried htmlspecialchars() function for cleaning text input but the problem is the same.
it because � chracther but i don't know how to remove it
![capture2](https://cloud.githubusercontent.com/assets/6761526/21470962/28f08162-cab4-11e6-9ce4-46fa03728855.PNG)
![capture](https://cloud.githubusercontent.com/assets/6761526/21470963/2a0830a4-cab4-11e6-800f-af7065fece04.PNG)

**Comment by Pashaster12** [comment]
@Blackops404 what about your charset?
